### PR TITLE
Fix missing currency in behat test

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix failing behat test by adding a missing currency to the context
| Type?         | bug fix
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | Travis must be green.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/15912)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

File: tests/Integration/Behaviour/Features/Context/Domain/CartFeatureContext.php [hunk 1]
```diff
@@ -42,6 +43,21 @@
...
 class CartFeatureContext extends AbstractDomainFeatureContext
 {
+    /**
+     * @given the current currency is :currencyIsoCode
+     */
+    public function addCurrencyToContext($currencyIsoCode)
+    {
+        $currency = new Currency();
+        $currency->name = $currencyIsoCode;
+        $currency->precision = 2;
```
Comment (Progi1984) [inline tests/Integration/Behaviour/Features/Context/Domain/CartFeatureContext.php:53]: @matthieu-rolland The precision is always 2 ?

File: tests/Integration/Behaviour/Features/Scenario/Cart/Calculation/Currency/currency.feature [hunk 2]
```diff
@@ -90,7 +90,7 @@ Feature: Cart calculation with currencies
     Given currency "currency2" is the current one
     Given there is a product in the catalog named "product1" with a price of 19.812 and 1000 items in stock
     When I add 3 items of product "product1" in my cart
-    Then my cart total should be 83 tax included
+    Then my cart total should be 83.0 tax included
     Then my cart total using previous calculation method should be 83 tax included
```
Comment (PierreRambaud) [inline tests/Integration/Behaviour/Features/Scenario/Cart/Calculation/Currency/currency.feature]:
```suggestion
    Then my cart total using previous calculation method should be 83.0 tax included
```

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Something must be change in a test

File: tests/Integration/Behaviour/Features/Context/Domain/CartFeatureContext.php (hunk 1 above)
Comment (matthieu-rolland) [reply tests/Integration/Behaviour/Features/Context/Domain/CartFeatureContext.php:53]: For now yes, this method is only called in one place, if someone want to set another precision for the same feature test (which is unlikely for now), he can still change the method and make it go through a parameter.

**Comment by matthieu-rolland** [comment]
> Something must be change in a test

@PierreRambaud done !

File: tests/Integration/Behaviour/Features/Context/Domain/CartFeatureContext.php (hunk 1 above)
Comment (Progi1984) [reply tests/Integration/Behaviour/Features/Context/Domain/CartFeatureContext.php:53]: :ok_hand: No problem. I validate.

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @matthieu-rolland

**Pull request merged**